QEfficient/generation/text_generation_inference.py

```diff
@@ -157,9 +157,8 @@ def get_compilation_dims(qpc_path: str) -> Tuple[int, int]:
 
 
 def get_input_prompts(prompt: str, prompts_txt_file_path: str) -> List[str]:
-    assert (
-        prompt is not None or prompts_txt_file_path is not None
-    ), "Please pass at least one argument either using --prompt or --prompts_txt_file_path"
+    if (prompt == None and prompts_txt_file_path == None):
```

Comment:
This should be- if (prompt == None **or** prompts_txt_file_path == None): as we ae checking either of condition in above statement.